app/controllers/pages_controller.rb

```diff
@@ -8,9 +8,9 @@ def home; end
   def index
     @incomes = Income.first(5)
     @expenses = Expense.first(5)
-    @expenses_categories = Category.joins(:expenses).group('title').sum('expenses.value')
-    current_expenses_value = Expense.sum(:value)
-    current_budget_value = Budget.last.value
+    @expenses_categories = Category.joins(:expenses).group('title').sum('expenses.value_cents')
+    current_expenses_value = Expense.sum(:value_cents)
```

Comment:
IMHO, `Expense` should `belong_to` `Budget`. WDYT @sleaz0id? 😜

Reply:
Now we can sum all of the expenses from the current_user's budget instead of all the expenses in the application :wink: